README.md

```diff
@@ -4,8 +4,153 @@ Measurement library will provide an open source alternative for sites/apps to se
 
 **This is not an officially supported Google product.**
 
-# Running tests and creating the build
 
+[![Build Status](https://travis-ci.org/googleinterns/measurement-library.svg?branch=master)](https://travis-ci.org/github/googleinterns/measurement-library/branches)
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
+![Contributions welcome](https://img.shields.io/badge/contributions-welcome-orange.svg)
+
+## Table of contents
+- [Overview](#overview)
+- [Installation](#installation)
+- [Usage](#usage)
+    - [Event Command](#event-command)
+    - [Set Command](#set-command)
+- [Event Processors](#event-processors)
+    - [Google Analytics](#google-analytics)
+    - [Custom Event Processor](#custom-event-processor)
+- [Storage Interfaces](#storage-interfaces)
+    - [Cookies](#cookies)
+    - [Custom Storage Interface](#custom-storage-interface)
+- [Local Setup](#local-setup)
+    - [Creating the build](#creating-the-build)
+    - [Running Tests Interactively](#running-tests-interactively)
+
+# Overview
+Measurement Library is a utility to help developers send data collected from their website to
+other libraries for analytics. After specifying settings in a script tag, you can process any
+sent events using a variety of built in event processors (e.g. sending data to Google Analytics).
+
+Data about the user is automatically saved to persist between visits using your choice of
```

Comment:
"Data about the user" makes it sound to me like it's building a profile or something, which is not the case for the GA processor.  I'd offer an alternative phrasing:

"Data from events or generated by event processors (is automatically saved...)"